terasoluna-gfw-validator/src/main/java/org/terasoluna/gfw/common/validator/constraints/Compare.java

```diff
@@ -79,6 +79,12 @@
     Operator operator();
 
     /**
+     * @return {@code true} it's valid if left and right are both null, but it's invalid if either is null. otherwise it's valid

```

Comment:
@ikeyat 
Thanks for your comments.
I've reflected in the javadoc. Please show 668bc68f0b7624153796f2027c242c2ce017a3e8.